tests/features/lru_cache_test.py

```diff
@@ -71,14 +96,170 @@ def test_fix_no_arg_decorators_noop(s, min_version):
             '@functools.lru_cache()\n'
             'def foo():\n'
             '    pass\n',
+            (3, 8),
             'import functools\n\n'
             '@functools.lru_cache\n'
             'def foo():\n'
             '    pass\n',
             id='call with attr',
         ),
+        pytest.param(
```

Comment:
these should be new separate tests

I also think you have too many tests that aren't covering the actual cases that matter

you want:

- noop: `from functools import lru_cache` (pyupgrade doesn't add or remove imports or change names because symbol tracking is hard)
- noop: `@functools.lru_cache(maxsize=10)` (unrelated parameter)
- positive: `@functools.lru_cache(maxsize=None)`


all the other tests are not helpeful